File: src/test/java/com/owncloud/android/UtilityTest.java
import org.junit.Test;
import static org.junit.Assert.*;

public class UtilityTest {

    @Test
    public void testAddition() {
        assertEquals(5, Utility.add(2, 3));
    }

    @Test
    public void testDivisionByZero() {
        try {
            Utility.divide(4, 0);
            fail("Expected an ArithmeticException to be thrown");
        } catch (ArithmeticException e) {
            assertEquals("/ by zero", e.getMessage());
        }
    }
}

class Utility {
    static int add(int a, int b) {
        return a + b;
    }

    static int divide(int a, int b) {
        return a / b;
    }
}

File: src/test/java/com/owncloud/android/UtilityTest.kt
import org.junit.Test
import kotlin.test.assertEquals
import kotlin.test.assertFailsWith

class UtilityTest {

    @Test
    fun testAddition() {
        assertEquals(5, Utility.add(2, 3))
    }

    @Test
    fun testDivisionByZero() {
        val exception = assertFailsWith<ArithmeticException> {
            Utility.divide(4, 0)
        }
        assertEquals("/ by zero", exception.message)
    }
}

object Utility {
    fun add(a: Int, b: Int): Int {
        return a + b
    }

    fun divide(a: Int, b: Int): Int {
        return a / b
    }
}
